.github/CODEOWNERS

```diff
@@ -596,6 +596,12 @@ app/uploaders/vets_shrine.rb @department-of-veterans-affairs/va-api-engineers @d
 app/validators/token_util.rb @department-of-veterans-affairs/lighthouse-banana-peels
 app/sidekiq/account_login_statistics_job.rb @department-of-veterans-affairs/vsp-identity
 app/sidekiq/bgs @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/sidekiq/bgs/flash_updater.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
Is this correct?

Reply:
I guess not, it looks like this had come in from one merge, and then it was removed later in master so now this is left over from a merge conflict